andela/lotus-ah-frontend
# fix(landing-page styling): fix get started button and singn up displa…

#### What does this PR do?
fix styling on the landing page

#### Description of Task to be completed?
- fix get the started button on the landing page
- fix singup form styling

#### How should this be manually tested?
```steps```
- start the app
- click on get started on the landing the page
- on the next page you should see  the second screenshort

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
#161983879

#### Screenshots (if appropriate)
<img width="1680" alt="screen shot 2018-11-15 at 3 56 39 pm" src="https://user-images.githubusercontent.com/24890605/48560866-17449b00-e8ef-11e8-9579-faea0cdcb22b.png">
<img width="1680" alt="screen shot 2018-11-15 at 3 56 47 pm" src="https://user-images.githubusercontent.com/24890605/48560962-3e02d180-e8ef-11e8-8ae5-ee901fe16d46.png">

#### Questions:
